packages/filezilla.rb

```diff
@@ -45,7 +45,7 @@ def self.build
   def self.install
     system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/env.d/"
-    filezilla = <<~'EOF'
+    filezilla = <<~EOF
       alias filezilla="WAYLAND_DISPLAY=wayland-0 DISPLAY='' GDK_BACKEND=wayland filezilla"
     EOF
     File.write("#{CREW_DEST_PREFIX}/etc/env.d/10-filezilla", filezilla)
```

Comment:
We can use FileUtils.install here too, yes?

Reply:
Done